### PR TITLE
#166108989 Response to Feedback

### PR DESCRIPTION
**Pull request by femitj**
**What does this PR do?**
This PR shows the implementation of feedback from LFA

**Description of Task to be completed?**
Have the following done:

implemented the following feedback:

 - clients cannot post loan application until status is verified
 - Float monetary values to significant of two, NO-ROUNDUP
 - CreatedOn should have DATE in sql
 - move test inside server dir
 - app.js should have response in json() not send()
 - middlewares should act like middlewares(move actions to service directory)
 - add query validations for get all repaid and all current loans
 - add error message if token expires

**How should this be manually tested?**

 - After you clone the repo,
 - cd into it and run npm install to install node modules.
 - Make sure you have nodejs and express installed.
 - Afterwards, run npm start to initiate the app.
 - test endpoint above using postman with localhost:8080 as the host
 - test endpoint above using postman with https://quick-credit.herokuapp.com/ as the host

**What are the relevant pivotal tracker stories?**
#166108989

**Screenshots**

Screenshot (46)

Screenshot (48)

Screenshot (47)

### CODE REVIEW

File: server/middleware/loan-middleware.js [hunk 1]
```diff
@@ -1,92 +1,72 @@
...
+dotenv.config();
+
 const middleware = {
-  async checkEligibility(req, res, next) {
-    const { rows } = await db(queries.checkRepaid(req.user.email));
+  async checkVerified(req, res, next) {
+    const { rows } = await db(queries.checkVerification(req.user.email, 'unverified'));
     if (rows.length) {
       return res.status(409).json({
         status: 409,
```
Comment (oluseyianani) [inline server/middleware/loan-middleware.js]: I'm not sure this is a 409. The user is trying to make an action he/she is unauthorized to make at the moment.

**Review by oluseyianani — COMMENTED** [review]
LGTM! Please fix the error code I've commented on and then you can merge. Good Job @femitj

File: server/middleware/loan-middleware.js (hunk 1 above)
Comment (femitj) [reply server/middleware/loan-middleware.js]: Yes, thank you i'll make changes to that

Comment (femitj) [reply server/middleware/loan-middleware.js]: Thank you for noting that, I've implemented it...

**Pull request merged**